week3/homework/src/actions/markAsDone.js

```diff
@@ -0,0 +1,22 @@
+const readTodosFile = require('./readTodosFile');
+const saveTodos = require('./saveTodos');
+
+
+
+async function markAsDone(req, res) {
+    try {
+        const todos = await readTodosFile();
+        const selectTodo = todos.find(todo => todo.id == req.params.id);
+        if (selectTodo == null) {
+            res.send(`There is no todo with id - ${req.params.id}`);
```

Comment:
what is the status that we use when something is not found ?
![image](https://user-images.githubusercontent.com/3776580/67893014-16cfff00-fb56-11e9-8ccf-31746905cb83.png)